model/group_test.go

```diff
@@ -57,6 +58,12 @@ func TestGroupFromReader(t *testing.T) {
 			Version:     "version",
 			CreateAt:    10,
 			DeleteAt:    20,
+			MattermostEnv: EnvVarMap{
```

Comment:
do we have test cases that the Mattermost env is nil or in a bad shape?

Reply:
Now we do :)

Reply:
(and we already had them in the `store` package, as I only added the MattermostEnv to 1 of the groups that gets tested there, so most of the test cases have MattermostEnv as `nil` actually)